docs/advanced-usage/connecting-cloud-storage/s3-compatible.md

```diff
@@ -1,11 +1,14 @@
 ---
+icon: simple/amazons3
```

Comment:
There is a minIO icon. We may want to use that here instead so this icon remains reserved for the s3 integration type

Reply:
Updated 👍 

Also added Hitachi as a compatible service — does this look good to you?